imagine-dragons-web/resources/css/songs-stylesheet.css

```diff
@@ -0,0 +1,20 @@
+#col1, #col2, #col3, #col4 {
+    padding-right: 5vw;
```

Comment:
The `vh`  and `vw` units have their use cases (why?). But you are abusing it in your stylesheets. Use `em`s instead and percentages only if needed. (replace all `vh` and `vw`)!